### PR TITLE
Analyzer deps bump

### PR DESCRIPTION
**Pull request by omri374**

### CODE REVIEW

**Comment by SharonHart** [comment]
@omri374 Build is now passing. 
Eventually I've realized that the timeout doesn't occur on newly uploaded model versions.
So upgrading to spacy 3.2.0 actually helped with model downloading in analyzer. 
What's left is image-redactor that resolves spacy version from analyzer (which is not yet bumped). The workaround of installing 3.0.0 was to install from tar.gz.
After merging and releasing, we can revert the tar.gz installation.

File: presidio-analyzer/Pipfile [hunk 1]
```diff
@@ -4,17 +4,17 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-spacy = "==3.0.6"
+spacy = "==3.2.0"
```
Comment (omri374) [inline presidio-analyzer/Pipfile:7]: Can we allow more flexibility here?

File: presidio-analyzer/setup.py [hunk 2]
```diff
@@ -29,14 +29,13 @@
...
-    tests_require=["pytest", "flake8==3.7.9"],
+    tests_require=["pytest", "flake8>=3.7.9"],
     install_requires=[
-        "spacy==3.0.6",
-        "regex>=2020.11.13",
-        "tldextract==3.1.0",
-        "pyyaml==5.4.1",
-        "pydantic==1.7.4",
-        "phonenumbers==8.12.24",
+        "spacy==3.2.0",
```
Comment (omri374) [inline presidio-analyzer/setup.py:34]: here too

**Review by navalev — DISMISSED** [review]

File: presidio-analyzer/Pipfile (hunk 1 above)
Comment (SharonHart) [reply presidio-analyzer/Pipfile:7]: I thought that for the dependencies changing the detections regularly, at least in the previous bumps we did, to lock it to minor.

Comment (SharonHart) [reply presidio-analyzer/Pipfile:7]: Anyway, done

**Review by SharonHart — APPROVED** [review]

**Review by navalev — APPROVED** [review]

**Pull request merged**